node/node.go

```diff
@@ -767,7 +767,7 @@ func (n *Node) initVMs() error {
 	// If staking is disabled, ignore updates to Subnets' validator sets
```

Comment:
nit: again here in the comment, I'd replace staking with sybil protection

Reply:
renamed